internal/worker/storageregistry/worker.go

```diff
@@ -227,20 +227,17 @@ func (w *storageRegistryWorker) workerFromCache(namespace string) (storage.Provi
 }
 
 func (w *storageRegistryWorker) initStorageRegistry(namespace string) error {
+	runner := providertracker.ProviderRunner[storage.ProviderRegistry](w.cfg.ProviderFactory, namespace)
```

Comment:
This is just a drive by right?

Reply:
Yes, I was going through the code and auditing what was using what, and I noticed I wasn't following the common practice :facepalm: